### PR TITLE
CLOUD-974 add instance check function

### PR DESCRIPTION
**Pull request by svenwltr**
> https://jira.rebuy.de/browse/CLOUD-974

Copypasta and a bit of new code.

### CODE REVIEW

**Comment by svenwltr** [comment]
build it

**Comment by svenwltr** [comment]
@rebuy-de/prp-k8s-ingress-elb-check Please review.

File: main.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	"github.com/aws/aws-sdk-go/service/elb"
+)
+
+const StateInService = "InService"
+
+func main() {
+	var err error
+
+	if len(os.Args) != 2 {
+		fmt.Printf("USAGE: %s <load-balancer-name>", os.Args[0])
```
Comment (bjoernhaeuser) [inline main.go:18]: os.Exit missing?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	sess := session.Must(session.NewSession())
+
+	instanceID, err := getInstanceID(sess)
+	err = checkInstance(sess, loadBalancerName, instanceID)
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	fmt.Printf("Instance '%s' of LoadBalancer '%s' is InService.\n", instanceID, loadBalancerName)
```
Comment (bjoernhaeuser) [inline main.go:33]: os.Exit(0)?

**Review by bjoernhaeuser — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (svenwltr) [reply main.go:33]: It's always `0`, if it exits normally.

File: main.go (hunk 1 above)
Comment (svenwltr) [reply main.go:18]: fixed

**Review by stephanlindauer — APPROVED** [review]

**Pull request merged**